lib/atom-types.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import type { Range } from 'atom'
+
+type Buffer = {
+    characterIndexForPosition: (position: Range) => number,
+    positionForCharacterIndex: (index: number) => any,
+    getTextInRange: (range: Range) => string,
+}
+
+type Point = {
+    toArray: () => [number, number]
+}
+
+export type TextEditor = {
```

Comment:
There isn't an `atom.TextEditor`, so I thought this solution is a little better than adding it to the `declare module 'atom'` in `flow-typed/modules.js`.